parsers/px-to-rem/README.md

```diff
@@ -0,0 +1,66 @@
+# PX TO REM
+## Description
+
+Convert pixel measurement to rem.
+
+## Interface
+
+```ts
+interface parser {
+  "name": "px-to-rem",
+  "options": {
+    "baseFontSize"?: number;
```

Comment:
I would rename this parameter as `basePixelValue` since this parser could be applied for something else than a font size transformation.